pkgs/applications/virtualization/xen/4.10.nix

```diff
@@ -8,6 +8,9 @@
 , withOVMF ? false, OVMF
 , withLibHVM ? true
 
+# xen
+, lvm2, ncurses, python2Packages
```

Comment:
Can we use python3?

Reply:
Xen 4.10 only supports Python 2. This line is moved from `generic.nix` to make that file python version agnostic.